### PR TITLE
[ign-transport8] Update BitBucket links

### PR DESCRIPTION
**Pull request by chapulina**
Signed-off-by: Louise Poubel <louise@openrobotics.org>

### CODE REVIEW

File: tutorials/02_installation.md [hunk 1]
```diff
@@ -67,7 +67,7 @@ versions are welcome.
 This installation procedure uses pre-compiled binaries in a local
 workspace. To make things easier, use a MinGW shell for your editing
 work (such as the [Git Bash Shell](https://msysgit.github.io/) with
-[Mercurial](http://tortoisehg.bitbucket.org/download/index.html)), and
+[Mercurial](http://tortoisehg.github.com/download/index.html)), and
```
Comment (maryaB-osr) [inline tutorials/02_installation.md:70]:
```suggestion
[Git](https://gitforwindows.org/)), and
```
Not sure about this one @chapulina

File: tutorials/07_relay.md [hunk 2]
```diff
@@ -22,16 +22,16 @@ cd ~/ign_transport_tutorial/docker
 
 We'll use Docker to configure the environment for this example. Feel free to
 install Docker following any of the existing guides available
-([here](https://bitbucket.org/osrf/vrx/wiki/tutorials/installDocker)'s one).
+([here](https://github.com/osrf/vrx/wiki/tutorials/installDocker)'s one).
```
Comment (maryaB-osr) [inline tutorials/07_relay.md:25]: @chapulina can't find a link to replace this dead one

File: tutorials/07_relay.md [hunk 3]
```diff
@@ -22,16 +22,16 @@ cd ~/ign_transport_tutorial/docker
...
 
 We're going to build a Docker image and run it inside your host computer.
-Download the [build.bash](https://bitbucket.org/ignitionrobotics/ign-transport/raw/ign-transport8/example/docker/build.bash), [run.bash](https://bitbucket.org/ignitionrobotics/ign-transport/raw/ign-transport8/example/docker/run.bash) and
-[Dockerfile](https://bitbucket.org/ignitionrobotics/ign-transport/raw/ign-transport8/example/docker/ign-transport/Dockerfile) files.
+Download the [build.bash](https://github.com/ignitionrobotics/ign-transport/raw/ign-transport8/example/docker/build.bash), [run.bash](https://github.com/ignitionrobotics/ign-transport/raw/ign-transport8/example/docker/run.bash) and
+[Dockerfile](https://github.com/ignitionrobotics/ign-transport/raw/ign-transport8/example/docker/ign-transport/Dockerfile) files.
```
Comment (maryaB-osr) [inline tutorials/07_relay.md:29]:
```suggestion
Download the [build.bash](https://github.com/ignitionrobotics/ign-transport/raw/ign-transport8/docker/build.bash), [run.bash](https://github.com/ignitionrobotics/ign-transport/raw/ign-transport8/docker/run.bash) and
[Dockerfile](https://github.com/ignitionrobotics/ign-transport/raw/ign-transport8/docker/ign-transport/Dockerfile) files.
```

File: tutorials/02_installation.md (hunk 1 above)
Comment (chapulina) [reply tutorials/02_installation.md:70]: Nice catch, I had been removing the `with Mercurial` bits, but I don't have a strong opinion either way.

File: tutorials/07_relay.md (hunk 2 above)
Comment (chapulina) [reply tutorials/07_relay.md:25]: We haven't ported VRX to GitHub yet, but my suggestion would be to point to an official Docker tutorial instead

Comment (maryaB-osr) [inline tutorials/07_relay.md:25]:
```suggestion
([here](https://docs.docker.com/get-docker/)'s one).
```

**Pull request merged**